src/api/controllers/v2/enrollment.controller.ts

```diff
@@ -21,15 +20,19 @@ export const createEnrollment = async (
   try {
     // check if user_id has the right role
     const payload: EnrollmentAttributes = {
-      userId: req.body.userId,
+      userId: req.auth.userId,
```

Comment:
I don't think `req.auth.userId` works here - is this route for manually adding instructors to courses? It should use `req.body.userId` imo, it seems wrong to have someone use `createEnrollment` for themselves like that

Reply:
that makes sense